models/food.js

```diff
@@ -6,7 +6,13 @@ var Food = {
   all: database.raw('SELECT * FROM foods')
   .then(foods => {
     return foods.rows
-  })
+  }),
+  find: function(id) {
+    return database.raw('SELECT * FROM foods WHERE id = ?', [id])
```

Comment:
you could use a Where here:
database.where('id', 1)
Outputs:
select * from `users` where `id` = 1